bundle/build

```diff
@@ -85,9 +90,40 @@ if [[ $ERRORED == 1 ]]; then
     exit 1
 fi
 
+# Local assets
 cp Makefile $TMPDIR
 cp README.md $TMPDIR
 
+curl_to() {
+    curl -sfL -o "$1" "$2"
+}
+TMP_BIN=$TMPDIR/bin
+
+# conmon
+curl_to $TMP_BIN/conmon \
+    https://github.com/containers/conmon/releases/download/$CONMON_VERSION/conmon
+chmod +x $TMP_BIN/conmon
+
+# runc
+curl_to $TMP_BIN/runc \
```

Comment:
the same here.

Reply:
See https://github.com/cri-o/cri-o/pull/3345#discussion_r389588866